apps/web/modules/insights/insights-virtual-queues-view.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import { useState } from "react";
+
+import { TestForm } from "@calcom/app-store/routing-forms/components/SingleForm";
+import type { RoutingForm } from "@calcom/app-store/routing-forms/types/types";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc";
+import { Label, Select } from "@calcom/ui";
+
+import InsightsLayout from "./layout";
+
+export default function InsightsVirtualQueuesPage() {
+  const { t } = useLocale();
+  const { data: routingForms, isLoading: isRoutingFormsLoading } =
+    trpc.viewer.insights.getUserRelevantTeamRoutingForms.useQuery();
```

Comment:
This only returns routing forms of teams the user is a member of + that have at least one route that redirects to a weighted round robin event type where the user is a host of